src/sync.js

```diff
@@ -459,7 +461,18 @@ SyncApi.prototype.sync = function() {
         // no push rules for guests, no access to POST filter for guests.
         self._sync({});
     } else {
-        getPushRules();
+        // Don't do an HTTP hit to /sync. Instead, load up the persisted /sync data,
+        // if there is data there.
+        client.store.getSavedSync().then((savedSync) => {
+            // Indicate whether we should sync after getting push rules,
```

Comment:
I don't understand this comment. Indicate how, to whom?

Reply:
Oh sorry, that was from a previously removed impl.